cmd/session_test.go

```diff
@@ -35,9 +35,9 @@ func (s *TestSuite) TestSession(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(isSessionDirExists(), Equals, true)
 
-	session := newSessionV8()
+	session := newSessionV8(getHash("cp", []string{"mybucket", "myminio/mybucket"}))
 	c.Assert(session.Header.CommandArgs, IsNil)
-	c.Assert(len(session.SessionID), Equals, 8)
+	c.Assert(len(session.SessionID) >= 8, Equals, true)
```

Comment:
How about the following assert? It asserts a more stronger property than `>= 8`. It uses the fact that the sha256 checksum part of the sessionID is 32 bytes. 
```suggestion
	c.Assert(len(session.SessionID) >= sha256.Size, Equals, true)
```

Reply:
This is not correct. Existing session support generates 8 bytes long random string whereas this PR generates guessable session ID using SHA-256 where the length is 32 bytes long. This change makes both to work.

When session support is deprecated and removed, this check will be fine-tuned.